.github/workflows/pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           java-version: 1.11
       - name: Compile and run tests
-        run: sbt clean +test
+        run: joern-cli/frontends/php2cpg/installdeps.sh; sbt clean +test
```

Comment:
maybe better to split into two `run` commands so that we can differentiate which one went wrong.